app/src/main/res/values-pl/strings.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
Please keep the xliff namespace here: `<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">` and keep the xliff usage as in the English strings.xml file.